Pilot/wasser.py

```diff
@@ -0,0 +1,157 @@
+"""
+   Wasser module is created for providing https requests for Python 2.6,
```

Comment:
I am still not convince with the name of this module. It is hard for the external users to understand the meaning.